cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -2845,6 +2854,9 @@ setRootTargets targetAction perPkgTargetsMap =
       | isSubLibComponentTarget tgt     = elab { elabHaddockInternal    = True }
       | otherwise                       = elab
 
+minVersionReplFlagFile :: Version
+minVersionReplFlagFile = mkVersion [3,9]
```

Comment:
Will need to be updated since this didn't make it into 3.10.1 afaict.

Reply:
Updated to 3,11.